microhab/SPEXpdd.tex

```diff
@@ -0,0 +1,254 @@
+
+%% *************************************************************************
+% LaTeX REFERENCES
+% ----------------
+%   Intro to LaTeX: http://www.rpi.edu/dept/arc/docs/latex/latex-intro.pdf
+%   Comprehensive LaTeX symbol list: http://tug.ctan.org/info/symbols/comprehensive/symbols-a4.pdf
+%% *************************************************************************
+
+\documentclass[conference]{IEEEtran} % http://www.ctan.org/pkg/ieeetran
+\usepackage[euler]{textgreek}
+\usepackage{blindtext}
+\usepackage{graphicx}
+\usepackage{nomencl}
+\usepackage{siunitx}
+\usepackage{hyperref}
+\usepackage[T1]{fontenc}
+\usepackage{etoolbox}
+\makenomenclature{}
+
+\title{\textmu{}Hab}
+
+
+\author {
+  \IEEEauthorblockN{% This block is for author Names.
+    Austin~Bodzas\IEEEauthorrefmark{1}
+  }
+  \IEEEauthorblockA{% This block is for the author Affiliations, aka department and university
+    RIT Space Exploration, Rochester Institute of Technology \\ %\\ starts a new line
+    Rochester, N.Y. \\
+    Email:
+    \IEEEauthorrefmark{1}abb6499@rit.edu
+  }
+}
+% page header for pages other than cover page
+\markboth{\textmu{}hab}%
+{Bodzas \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+
+      % The abstract is a brief summary of the design document. Typically it includes the purpose of the design document, key goals or objectives, and justifications.
+      % Be sure not to confuse the abstract with the introduction.
+      % It is easiest to write the abstract after the rest of the paper has been written.
+      % That way you can choose key information from the sections that you've already completed and string them together in the abstract.
+      % Consider the abstract to be your elevator pitch to anyone reading this design document.
+      % What are they reading?
+      % What is the goal?
+      % Why is it worth my time?
+      % The abstract is what will show up in Google results and other search engines, and what people will read when they are deciding what is worth their time and brain power.
+    A project definition for the \textmu{}Hab project.  \textmu{}Hab is the term the
+    HAB team has been using to describe a smaller balloon and payload than is
+    typically used by the group (1200 grams). However, this project expands upon
+    the term with the main goal of minimizing costs that go into building and
+    launching a HAB.\@
+\end{abstract}
+
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{%
+\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{}
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
+% Below are examples of using nomenclature for math symbols and constants or units
```

Comment:
add:
-HAB (High-Altitude Balloon)
-PCB (Printed Circuit Board)
-COTS (Commercial Off-The-Shelf)